portal-web/docroot/html/common/themes/password_expiring_soon.jspf

```diff
@@ -14,13 +14,13 @@
  */
 --%>
 
-<c:if test="<%= themeDisplay.isSignedIn() && UserLocalServiceUtil.isPasswordExpiringSoon(user) %>">
-	<aui:script use="liferay-notice">
-		new Liferay.Notice(
+<c:if test='<%= SessionMessages.contains(request, "passwordExpiringSoon") %>'>
```

Comment:
@jonmak08 looks like the source formatter is complaining about one of the changes related to this pull - I think line 110 of LoginPostAction needs one more tab.